src/services/tx/txSender.ts

```diff
@@ -71,7 +71,7 @@ export const createTx = async (txParams: SafeTransactionDataPartial, nonce?: num
 export const createMultiSendTx = async (txParams: MetaTransactionData[]): Promise<SafeTransaction> => {
   const safeSDK = getAndValidateSafeSDK()
   const tx = await safeSDK.createTransaction(txParams)
-  return createTx({ ...tx.data, operation: 1 })
+  return createTx({ ...tx.data })
```

Comment:
You can just pass `tx.data` without spreading.

Reply:
In fact, you could now simply delete the function and use `createTx` instead.

Reply:
`createTx` doesn't support passing an array though so we would have to change it in a few places and also handle getting the recommended nonce. I suggest we do this refactoring as a separate task. I removed the spreading

Reply:
OK, gotcha. Then please at least rewrite it like this:
```
export const createMultiSendTx = async (txParams: MetaTransactionData[]): Promise<SafeTransaction> => {
  return withRecommendedNonce((safeSDK) => safeSDK.createTransaction(txParams))
}
```

Reply:
Ah makes sense, thanks!